nemo/collections/nlp/modules/common/megatron/transformer.py

```diff
@@ -213,8 +214,14 @@ def __init__(
                 self.input_layernorm = LayerNorm1P(
                     hidden_size, layernorm_epsilon, sequence_parallel_enabled=sequence_parallel
                 )
+            elif normalization == 'low_precision_layernorm':
+                self.input_layernorm = LPLayerNorm(hidden_size, layernorm_epsilon)
             else:
                 self.input_layernorm = MixedFusedRMSNorm(hidden_size, layernorm_epsilon)
+            if not bias:
+                for module in self.input_layernorm.modules():
+                    if hasattr(module, 'bias') and isinstance(module.bias, nn.Parameter):
+                        module.register_parameter('bias', None)
```

Comment:
Please add a comment explaining why we nee dthis

Reply:
Added and fixed